src/pkg/services/m365/api/access.go

```diff
@@ -0,0 +1,62 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/alcionai/clues"
+
+	"github.com/alcionai/corso/src/internal/m365/graph"
+)
+
+// ---------------------------------------------------------------------------
+// controller
+// ---------------------------------------------------------------------------
+
+func (c Client) Access() Access {
+	return Access{c}
+}
+
+// Access is an interface-compliant provider of the client.
+type Access struct {
+	Client
+}
+
+func (c Access) GetToken(
```

Comment:
Can we add a test for this? Also, is this documented by MSFT somewhere? Could you please leave the link as a comment?

Reply:
d'oh!  I did miss that, didn't I?  Thanks for the catch.